hazelcast/src/main/java/com/hazelcast/cache/impl/eviction/EvictionPolicyEvaluatorProvider.java

```diff
@@ -45,11 +45,11 @@ private static void init() {
     /**
      * Gets the {@link EvictionPolicyEvaluator} implementation specified with <code>evictionPolicy</code>.
      *
```

Comment:
I think we can move getEvictionPolicyEvaluator method into EvictionPolicyType enum as a static method and remove this EvictionPolicyEvaluatorProvider class.